### ISSUE TITLE
error: [Errno 2] No such file or directory: 'README.md'

### ISSUE THREAD

**Issue by donaldforest** [Codaone/DEXBot#19]
Ubuntu 17.10
python setup.py install

```
Writing /tmp/easy_install-7gmvfh06/click-datetime-0.2/setup.cfg
Running click-datetime-0.2/setup.py -q bdist_egg --dist-dir /tmp/easy_install-7gmvfh06/click-datetime-0.2/egg-dist-tmp-4pfgk04_
error: [Errno 2] No such file or directory: 'README.md'
```
```
~/DEXBot$ ls
app.py	app.spec  build  cli.py  config.yml  dexbot  dexbot.egg-info  dist  docs  hooks  LICENSE.txt  Makefile	MANIFEST.in  README.md	setup.cfg  setup.py
```

**Comment by mikakoi** [comment]
This is a problem in the click-datetime library, I found that if you just run setup twice, it will succeed on the 2nd try.

**Comment by donaldforest** [comment]
Yes, I run install more than 2ce. I fix couple of troubles with dependencies... But I runn couple of times after getting this error:
`error: [Errno 2] No such file or directory: 'README.md'`
How to fix this?

**Comment by svdev** [comment]
For me the following seems to work out:

```
pip install click-datetime PQt5
python setup.py install 
```

**Comment by mikakoi** [comment]
Fixed as of #22. Now you can use make to install dexbot. Or just simply install by hand with pip.
